website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -139,6 +141,15 @@ If `enable_auto_scaling` is set to `false`, then the following fields can also b
 
 * `node_count` - (Required) The number of nodes which should exist within this Node Pool. Valid values are between `0` and `1000`.
 
+---
+
+A `upgrade_settings` block exports the following:
+
+* `max_surge` - The maximum number or percentage of nodes which will be added to the Node Pool size during an upgrade.
```

Comment:
since this is configurable here, this wants to be:

```suggestion
* `max_surge` - (Required) The maximum number or percentage of nodes which will be added to the Node Pool size during an upgrade.
```